environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverter.java

```diff
@@ -224,9 +224,11 @@ private AzureParametersDto azureParamsToAzureParametersDto(AzureEnvironmentParam
 
     private AzureResourceEncryptionParametersDto azureResourceEncryptionParametersToAzureEncryptionParametersDto(
             AzureResourceEncryptionParameters azureResourceEncryptionParameters) {
-        return AzureResourceEncryptionParametersDto.builder()
-                .withEncryptionKeyUrl(azureResourceEncryptionParameters.getEncryptionKeyUrl())
-                .build();
+        AzureResourceEncryptionParametersDto.Builder azureResourceEncryptionParametersDto = AzureResourceEncryptionParametersDto.builder()
+                .withEncryptionKeyUrl(azureResourceEncryptionParameters.getEncryptionKeyUrl());
+        NullUtil.doIfNotNull(azureResourceEncryptionParameters.getEncryptionKeyResourceGroupName(), encryptionKeyResourceGroupName ->
+                azureResourceEncryptionParametersDto.withEncryptionKeyResourceGroupName(encryptionKeyResourceGroupName));
```

Comment:
Not a big deal but I do not think we need the null-safe execution. The DTO builder can happily accept `null` for `withEncryptionKeyResourceGroupName()`; see also `AzureEnvironmentParametersConverter.postConvertToDto()` below for a logic avoiding `doIfNotNull()`. The default value for `encryptionKeyResourceGroupName` is `null` anyway; setting it to `null` explicitly will not cause any harm. 😄